pages/analysis.md

```diff
@@ -1,15 +1,36 @@
+[Home](https://michaelrappa.github.io/usa-migration/) |
+[Analysis](https://michaelrappa.github.io/usa-migration/pages/analysis.html) |
+[Background](https://michaelrappa.github.io/usa-migration/pages/background.html) |
+[Reflections](https://michaelrappa.github.io/usa-migration/pages/reflections.html) |
+[Workflow](https://michaelrappa.github.io/usa-migration/pages/workflow.html) |
+[Credits](https://michaelrappa.github.io/usa-migration/pages/credits.html) |
+[Bibliography](https://michaelrappa.github.io/usa-migration/pages/bibliography.html) |
+[Sources](https://michaelrappa.github.io/usa-migration/pages/sources.html)
+
+
 # Analysis
 
-This is where you where you discuss any (initial) conclusions you have come to. This is also a natural place to put any and all visualizations that you come up with. In fact, this is where you tell the reader what it is you discovered. You can use the first person if that helps you discuss the process. This is where you both describe to the reader what they are looking at (if they are looking at a visualization) and, most importantly, what your interpretation of the results is.
+![In this image it shows the inflow of migrations from Europe from the year 1825 until 1935 to show how immigration started to decrease going into the United States Pre World War II.]({{ site.baseurl }}/assets/images/Pre WWII Immigration.png)
+
+This image shows the inflow of immigration to the USA from Europe from the years between 1825 and 1935, showing the diverse tapestry of imigrants from European nations. It also demonstrates that immigration started to decrease in the lead up to World War II becasuse of the economic downturn experiences at that time.
+
+![In this image it shows five major laws which are 1790 Naturalization Act, 1917 Immigration Act, 1953 Refugee Relief Act, 1965 Hart Celler Act, and 1990 immigration Act. All which have had a big effect to global migration to the United States.]({{ site.baseurl }}/assets/images/Immigration-Laws.png)
+
+This image shows five major laws influencing immigration to the USA which include the 1790 Naturalization Act, the 1917 Immigration Act, the 1953 Refugee Relief Act, the 1965 Hart Celler Act, and the 1990 immigration Act. Each of these laws had a demonstrable effect on imigration the the United States. 
+
+![In this image it shows Wars and events that have effected migrations to the United States. The wars and events are California Gold Rush, Irish Potato Famine, U.S. Civil War, Unification of Germany, Angell Treaty of 1880, World War I,The Great Depression, World War II, Guatamela Civil War, Honduras Wars & Curruption, and El Salvadorean Civil War. All which had caused an increase or a decrease of migration to the U.S.]({{ site.baseurl }}/assets/images/Major Events That Affected Immigration in the United States.png)
```

Comment:
Are these the only visuals? These are three individual worksheets, why not make use of dashboards? That way you could combine different visuals, and use other charts alongside line charts.